app/helpers/application_helper.rb

```diff
@@ -21,7 +21,7 @@ def controller_t(lazy_t, *args)
 
   def menu_button
     button_tag(t('generic.menu'),
-               type: 'button', role: 'button', class: 'govuk-header__menu-button js-header-toggle',
+               type: 'button', role: 'button', class: 'govuk-header__menu-button js-header-toggle no-print',
```

Comment:
The only place I can find `menu_button` being used is within `app/views/layouts/_header.html.erb`:
```html
    <div class="govuk-header__content">
      <%= home_link %>
      <%= menu_button if user_header_link %>
      <nav>
        <%= user_header_link %>
      </nav>
    </div>
```
And to be honest, I'm surprised that whole block isn't `no-print` rather than just this element.

If the layout is problematic if there isn't a `.govuk-header__content`, perhaps an inner `.no-print` `div` around the links would do it.